cloudstate/function_servicer.py

```diff
@@ -0,0 +1,179 @@
+"""
+Lightbend, TJS, Apache 2.0 2020
+"""
+
+import logging
+from typing import List
+
+import grpc
+from google.protobuf import symbol_database as _symbol_database
+from google.protobuf.any_pb2 import Any
+from grpc._server import _RequestIterator
+
+from cloudstate.entity_pb2 import ClientAction
+from cloudstate.function_pb2 import FunctionCommand, FunctionReply
+from cloudstate.function_pb2_grpc import StatelessFunctionServicer
+from cloudstate.stateless_function_context import StatelessFunctionContext
+from cloudstate.stateless_function_entity import (
+    StatelessFunction,
+    StatelessFunctionHandler,
+)
+
+_sym_db = _symbol_database.Default()
+
+TYPE_URL_PREFIX = "type.googleapis.com/"
+
+
+def get_payload(command):
```

Comment:
I see some duplications between this class and the CloudstateEventSourcedServicer class that opens up possibilities for refactoring. Perhaps extracting to a utility class.